embrace-android-api/src/main/kotlin/io/embrace/android/embracesdk/internal/api/NetworkRequestApi.kt

```diff
@@ -15,7 +15,7 @@ public interface NetworkRequestApi {
      */
     public fun recordNetworkRequest(networkRequest: EmbraceNetworkRequest)
 
-    public fun getTraceIdHeader(): String
+    public val traceIdHeader: String
```

Comment:
Given that we have NSF as well as OTel stuff to consider, this naming is pretty problematic. 

We shouldn't change it in this PR, but we should probably change this when we bump our SDK version to 7.0